src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -81,6 +81,8 @@
     private boolean allow_merge_commit;
     private boolean allow_rebase_merge;
 
+    private boolean delete_branch_on_merge;
```

Comment:
This can be called `deleteBranchOnMerge` now and it should still work. 
```suggestion
    private boolean deleteBranchOnMerge;
```